prow/oss/kubeconfigs.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+clusters:
+- cluster:
+    certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUVMVENDQXBXZ0F3SUJBZ0lSQUpHVjdxckthRDBXK0pUcFJDYVYrMUV3RFFZSktvWklodmNOQVFFTEJRQXcKTHpFdE1Dc0dBMVVFQXhNa01EZG1PR1kxTTJRdE1qRTBZUzAwTjJRNUxXSTVNV010T0dOa01HRmpOREprTnpNNApNQ0FYRFRJek1Ea3hNVEU1TVRJeU4xb1lEekl3TlRNd09UQXpNakF4TWpJM1dqQXZNUzB3S3dZRFZRUURFeVF3Ck4yWTRaalV6WkMweU1UUmhMVFEzWkRrdFlqa3hZeTA0WTJRd1lXTTBNbVEzTXpnd2dnR2lNQTBHQ1NxR1NJYjMKRFFFQkFRVUFBNElCandBd2dnR0tBb0lCZ1FDdTRRNnlwYzUrc2xyUkR3UXkvRVBhSmFWTzBWT1BEWlZuVTlULwptUGhrZ0hoZk94SUZSZTk2V2N6eVdZU1AxN1h6b3Q1ZVpaVUY0bDFnSlhkVEdpTG5wUk50SjZobkNYMXNudTZxCnJSelU0cm5LcnMrdDJzbHJneXdhVWgyQVFLbzlwakU5enJsRTNWT2Z6RG5WMkJwaklFV05wWTZHQktyYVBZeFoKb0xjbnhhRktjaHB2UHpZM0RvNTVZUE11S1R5R3NmYnFJbStnMXh1ZnJxWGI1MlB6U0J5UU9KdGtJdFE5TFJEbAp6bHZ5K0FOODZ2YUJmUzRYZmFWTm1MQlFsaGN3T2pJZjhpRXBPSngxaVRmUkg0MUs5L01GNk9yWFl1SElKa0YxCktUSmh1V1h5M1QyZDJGNmhGYzdSQ0YzR0Ntc0FEVDN4REw1UmRYVEdZcW1UUnlzcFp1VXZUclR4M1h1TkVUazAKWjFyYmsvOGdhV2ZBaytpMTg5MXp4UVd6dXVDZldIQXZiU1dyZWo3OG5sNnhIbk9XMjBnTklkRmpQbzJhSjM5aApMd0hnRkgvamk2SGxrTmFJL1RZWFFuQVZPaXZhRnA2Z2N3UU41NE5wZUlyVjVlMi9LUmVuVzNQQWtFSUtGa0JBCllNY2lQK1lNRTNsYzRVcFplTVhVYWJOamV0VUNBd0VBQWFOQ01FQXdEZ1lEVlIwUEFRSC9CQVFEQWdJRU1BOEcKQTFVZEV3RUIvd1FGTUFNQkFmOHdIUVlEVlIwT0JCWUVGTGJVNHRNTFlYSU5KdFBISDlVdGV1QXBNazFnTUEwRwpDU3FHU0liM0RRRUJDd1VBQTRJQmdRQi81OVp4aHN6S3FzbDFUZ1kwOGR5eC93aEFPbkd2LzFSOXJpTXJ1VDZkClZOZG1HTTR5eGFiMUtuZ2N5K1l5QTZqWnBoN0pCUUY2T2t1ZWY3Nll1NktlcjZmVFo4SWxrS3k4bGNtMmJWMHEKcnBlS01OcnQ5bGlmbkNhdUZFaGNoY2hyaElhTXgzWWd2M1FIU3Z2SEVnek8wWjlmUmtWZ0NjTTNnV0VPVXZvbQpEckt0WmExdng1NWtuUU5ScllPZ0ZJNVdtb2FqSzVGSzdCeUFodHFBV01VOXU4OTNkb3plZU9WZGUzNGt1Z0VqCkV4b2tJT3I5aGh0Tkl0eDhESld0c3lJN0dsNmhWVVlMMElBVlU1YU5IWDZQSVpCNnpXelJyN2pEMURPUzUwT3cKZThOT3FlYTY3MFFLOUNwcG02azF2TEF1NjlSRXhWbCtuMHVUcS9XejdENnF6dUpTMCtmZVBWdEJJV1BQNk54Qwp5b3ZOczlleGUxYUsrL0JjYkFqeVF5OSsycHR0MDlTbUNwa0JualVVWnNUVmV4TklxTkp0d1lCbzg5TjdBbjdKCmNPSnlidHNhaDJDYzBHelNLWU5QV1hranU5U1c4Ri9ZWmdnUTdIVTAzb05QNXpaZ2pNTjFkcXVwVzdvQnVBSloKTU8xaEhWdDNqeU8vZGtHclY1VktNMjA9Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K
+    server: https://35.185.203.10
+  name: gke_oss-prow_us-west1-a_prow
+contexts:
+- context:
+    cluster: gke_oss-prow_us-west1-a_prow
+    user: gke_oss-prow_us-west1-a_prow
+  name: gke_oss-prow_us-west1-a_prow
+current-context: gke_oss-prow_us-west1-a_prow
+kind: Config
+preferences: {}
+users:
+- name: gke_oss-prow_us-west1-a_prow
+  user:
+    exec:
+      apiVersion: client.authentication.k8s.io/v1beta1
+      args: null
```

Comment:
Need the application default credentials flag here `--use_application_default_credentials`

Reply:
Done in #2065